scripts/upgrade

```diff
@@ -29,6 +29,35 @@ if [ -z "${secret_key:-}" ]; then
 	ynh_app_setting_set --app=$app --key=secret_key --value=$secret_key
 fi
 
+#=================================================
+# KEEP CONFIGS IF NEEDED
+#=================================================
+
+if [ -f "$install_dir/config.local.yunohost.php" ]
+then
+	ynh_script_progression --message="cleaning conf files..." --weight=5
+	mv "$install_dir/config.local.php" "$install_dir/config.local.php.old"
+	mv "$install_dir/config.local.yunohost.php" "$install_dir/config.local.php"
+	chmod 650 "$install_dir/config.local.php"
```

Comment:
I think it would probably be better if we change the rights to be read-only.

```suggestion
	chmod 440 "$install_dir/config.local.php"
```

Reply:
Ok for these read only permissions, also for the `config.local.user.php`, isn't it ?

Reply:
> also for the config.local.user.php, isn't it ?

I don't think so, because we want the user local file to be editable by the user. So `600` or `660` (if the group for this file is `$app`) is fine.